### ISSUE TITLE
Mute SMS sender

### ISSUE THREAD

**Issue by Tealk** [you-apps/ConnectYou#345]
### Describe your suggested feature

A function to mute certain senders would be great, I live near the border and receive SMS from Telekom a few times a day because I switch to a non-German network.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Duplicate of #323

**Comment by Tealk** [comment]
not really, i don't want to block it, just don't want to get a notification.

**Comment by Tealk** [comment]
@Bnyro do you see the answers when the issue is closed?

**Comment by Bnyro** [comment]
Not planned then.
